CIP-Fair_Stakepool_Rewards/README.md

```diff
@@ -0,0 +1,195 @@
+---
+CIP: 75
+Title: Fair Stake Pool Rewards
+Status: Proposed
+Category: Reward-Sharing Scheme
+Authors:
+    - Tobias Fancee <tobiasfancee@gmail.com>
+Implementors:
+    - Input Output Global
+    - Cardano Foundation
+    - Emurgo
```

Comment:
Has any of those implementors agreed to be listed here? I hardly think so. As a reminder in CIP-0001:

> Implementors **must sign off on the plan** and be referenced in the document's preamble.

Note also that implementors must be individuals, not entities. Otherwise there's simply no accountability.

Reply:
Following on this it's become apparent at the CIP meeting today that this & all other RSS CIP's should only be merged with a status of `Inactive` (also noted in https://github.com/cardano-foundation/CIPs/pull/422#issuecomment-1385152152).